tests/foreman/destructive/test_capsulecontent.py

```diff
@@ -79,15 +80,106 @@ def test_positive_sync_without_deadlock(
     proxy.download_policy = 'immediate'
     proxy.update(['download_policy'])
 
+    nailgun_capsule = large_capsule_configured.nailgun_capsule
```

Comment:
I never understood why the capsule should have a nailgun pre-fix, I know this is coming from API. The capsule would be a capsule it's nothing to do with a nailgun.

Reply:
just making as `capsule = large_capsule_configured.nailgun_capsule` should be fine

Reply:
I think it'd better to be specific what capsule object is about, so I'd keep it as is

Reply:
> I never understood why the capsule should have a nailgun pre-fix, I know this is coming from API. The capsule would be a capsule it's nothing to do with a nailgun.

If I would be you, @omkarkhatavkar , I would ask @JacobCallahan about why did he named it `nailgun_`.
See 818bd5407f2ff5fd093f744029d1f0d54ed893b9

Reply:
> just making as `capsule = large_capsule_configured.nailgun_capsule` should be fine

see https://github.com/SatelliteQE/robottelo/pull/12022#discussion_r1285198467

Reply:
> I think it'd better to be specific what capsule object is about, so I'd keep it as is

If we wanted to be specific then it is a capsule object. Adding nailgun_prefix does not make sense to me at least. 

>If I would be you, @omkarkhatavkar , I would ask @JacobCallahan why he named it nailgun_.
See https://github.com/SatelliteQE/robottelo/commit/818bd5407f2ff5fd093f744029d1f0d54ed893b9

Thanks for the help there! (good learnings)